assets/csssoftvision.css

```diff
@@ -0,0 +1,44 @@
+body {
+  font: normal medium/1.4 sans-serif;
+    color:darkslategray;
+        
+}
+table {
+    border-collapse: collapse;
+    width: 100%;
+}
+
+th, td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid #ddd;
+}
+thead { 
+    color:white;
+    background-color: #00BFFF;
+    border-bottom-color:  #00BFFF;
+}
+
+<style>
```

Comment:
What is this poor thing still doing here? This is CSS, little tag, go home! Go on now, shoo! Get! :stuck_out_tongue: